From 48263a3e9e26f8cf1941d6cff7f6afa48952ebf2 Mon Sep 17 00:00:00 2001
From: Martin George <marting@netapp.com>
Date: Fri, 31 Jan 2025 22:00:29 +0530
Subject: [PATCH] udev-rules-ontap: switch to queue-depth iopolicy

The queue_depth based iopolicy suits ONTAP NVMe controllers better.
So switch to the same.

Signed-off-by: Martin George <marting@netapp.com>
---
 nvmf-autoconnect/udev-rules/71-nvmf-netapp.rules.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/nvmf-autoconnect/udev-rules/71-nvmf-netapp.rules.in b/nvmf-autoconnect/udev-rules/71-nvmf-netapp.rules.in
index 99b6a8ba..c76a3c43 100644
--- a/nvmf-autoconnect/udev-rules/71-nvmf-netapp.rules.in
+++ b/nvmf-autoconnect/udev-rules/71-nvmf-netapp.rules.in
@@ -1,5 +1,5 @@
-# Enable round-robin for NetApp ONTAP and NetApp E-Series
-ACTION=="add", SUBSYSTEM=="nvme-subsystem", ATTR{subsystype}=="nvm", ATTR{model}=="NetApp ONTAP Controller", ATTR{iopolicy}="round-robin"
+# Set appropriate iopolicy for NetApp ONTAP and NetApp E-Series
+ACTION=="add", SUBSYSTEM=="nvme-subsystem", ATTR{subsystype}=="nvm", ATTR{model}=="NetApp ONTAP Controller", ATTR{iopolicy}="queue-depth"
 ACTION=="add", SUBSYSTEM=="nvme-subsystem", ATTR{subsystype}=="nvm", ATTR{model}=="NetApp E-Series", ATTR{iopolicy}="round-robin"
 
 # Set ctrl_loss_tmo to -1 for NetApp ONTAP NVMe/TCP
-- 
2.43.5

